docs/templates/PACKAGE_MAIN_DOCUMENTATION.MD

```diff
@@ -0,0 +1,105 @@
+**📦 Resources**
+
+<!-- RESOURCES:
+* required
+
+Listing of resources related to the topic including running versions of used code snippets or other applications. 
+Resource links are prefixed with the "🔗" icon
+-->
+
+Example:
+> [🔗 GitHub](https://github.com/BioPhoton/rx-angular-cdk-coercing).
+
+
+# ⛽ Motivation
+
+<!-- MOTIVATION:
+* required
+
+Motivation is here to point out the value for the consumer. If possible we can make clear in which situations the impact will get visible most.
+
+Importent to cover in this section is:
+- Theoretical background to understand the problem
+- The problem and pain points in detail
+- Sketch a optimal solution for the problem
+-->
+
+Example:
+> 
+
+# Available Approaches
+<!-- AVAILABLE_APPROACHES:
+* required
+
+This section should sum up current available approaches and point out the mors importat differences.
+The listing can be done in a bullet point list with optional links.
+-->
+
+Example:
+> - Disabling zone.js completely (compiler options)
+> - Disabling specific API's (zone configuration)
+> - Overriding Angular dependencies (custom event manager)
+> - Using `unpatch`
+
+# 🎯 RxAngular Solution
+
+## Installation and Setup
+
+<!-- INSTALL_INSTRUCTIONS:
+* required
+
+Install istructions and code snippets/terminal commands to run. 
+-->
+
+Example:
+> ```bash
+> npm install --save @rx-angular/cdk
+> # or
+> yarn add @rx-angular/cdk
+> ```
+
+## Usage
+
+<!-- USAGE:
+* required
+
+[TODO]
+-->
+
+## Best Practices
+
+<!-- BEST_PRACTICES:
+* required
+
+This section should list the most important things to know when using the package. 
+
+The following things could be covered:
+- Good/Bad examples
+- Folder and file organization
+- Measurements
+
+-->
+----
+
+## Testing
+
+<!-- TESTING:
+* required
```

Comment:
I wouldn't make this one required. I cannot imagine a dedicated testing section for  `coercing` or `coalescing`. The behavior itself is tested on our end. What do you want to explain here?
I see the point for the `template` or `state` package where one would have to take special care of providers or such things.